formal-spec/Leios/SimpleSpec.agda

```diff
@@ -115,45 +121,54 @@ data _⇀⟦_⟧_ : Maybe LeiosState → LeiosInput → LeiosState × LeiosOutpu
        ────────────────────────────────────
        just s ⇀⟦ FTCH-LDG ⟧ (s , FTCH-LDG l)
 
-  -- TODO: Base chain
+  -- Base chain
+
+  Base₁ : ∀ {bs bs' eb txs} → let open LeiosState s in
+       ∙ eb ∈ filterˢ (λ eb → isVote2Certified s eb × eb ∈ᴮ slice L slot 2) (fromList EBs)
+       ∙ bs BF.⇀⟦ B.SUBMIT (inj₁ eb) ⟧ (bs' , B.EMPTY)
```

Comment:
Giorgos confirmed that the base chain ignores this if it doesn't make a block in the current slot. I think we should make at least a note of this somewhere.

Reply:
How about adding a predicate `canProduceRB` that checks that the ranking block can be added by the party?